apps/sms/views/inbox/js/inbox.js

```diff
@@ -251,6 +251,11 @@ var InboxView = {
       titleContainer.replaceChild(groupTitle, title);
     }
 
+    if (!title.textContent) {
+      // display something before waiting for findContact's result
+      title.textContent = threadNumbers[0];
+    }
+
```

Comment:
This reproduces the behavior we have in master. I find it more correct to set it in setContact so that everything is at the same place.

In the future I think we should try to do better especially for group mms. For example display something like "<n> recipients" while we resolve it .